fastly/datacenters.go

```diff
@@ -0,0 +1,24 @@
+package fastly
+
+// Datacenter is a list of Datacenters returned by the Fastly API.
+type Datacenter struct {
+	Code        string            `mapstructure:"code"`
+	Coordinates map[string]string `mapstructure:"coordinates"`
```

Comment:
What is the reasoning for this being a `map[string][string]` and not a new type `Coordinates` struct?

Reply:
Simplicity, I can add a new `Coordinates` struct that matches the datatypes returned by the API